### PR TITLE
Feature #2282 coord_release_docs

### LINKED ISSUE #2282: Documentation: Add coordinated release checklist to the METplus Release Guide
## Describe the Enhancement ##
During the Coordinated METplus Release 5.1 process, it became clear that some steps are duplicated across the components. Redoing or rechecking the same steps multiple times for each component is unclear and requires more coordination than is needed. This issue it review the official release instructions for the components and factor out any steps that can be moved into a Coordinated METplus Release checklist. These include creating the coordinated release announcement, removing development release announcements for all components from the DTC website, Zenodo version updates, sending an email to RAL-IT requesting the installation of the new versions in /usr/local, updating the information on the [MET Download page](https://dtcenter.org/community-code/model-evaluation-tools-met/download) under "Sample Script For Compiling External Libraries And MET" (if these instructions haven't moved to RTD), and perhaps others.

### Time Estimate ###
4 hours?

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
None needed.

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required: John HG with Julie and George as reviewers/conributors
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
No impacts.

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by JohnHalleyGotway**
## Pull Request Testing ##

Big picture, I defined what a "coordinated release" means and created instructions for creating a new one. This enabled me to remove a couple of the instructions from the component release instructions. I *think* these changes are in pretty good shape but will rely heavily on your review for completeness and edits.

- [x] Describe testing already performed for these changes:</br>
Rendered and reviewed docs locally to confirm they works as expected.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- @georgemccabe and @jprestop, please review the content of this PR:
https://metplus--2288.org.readthedocs.build/en/2288/Release_Guide/index.html

- @lisagoodrich, please advise on proposed formatting changes. I changed...
From Italics: "Click the *Save* button"
To Bold: "Click the **Save** button"
Per #1667, I'm not sure which it should be. My personal opinion is that bold sticks out better than italics, but please LMK what standard you've already defined, if any.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**
That's all it is.

- [x] Do these changes include sufficient testing updates? **[Yes]**
None needed.

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[Monday 8/7/23]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Contributors_Guide/github_workflow.rst
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst:1]: Line 66 should change from:

> (e.g. **METplus-5.1.0 Development**)

to

> (e.g. **METplus-Wrappers-5.1.0 Development**)

File: docs/Release_Guide/index.rst
Comment (georgemccabe) [inline docs/Release_Guide/index.rst]:
```suggestion
An official release is a stable release of a METplus component and typically matches
```

File: docs/Release_Guide/index.rst
Comment (georgemccabe) [inline docs/Release_Guide/index.rst]:
```suggestion
#. **Official Release** (e.g. vX.Y.0) from the develop branch (becomes the new main_vX.Y branch)
```

File: docs/Release_Guide/coordinated.rst
Comment (georgemccabe) [inline docs/Release_Guide/coordinated.rst]:
```suggestion
bugfix component releases have already been created. Note, however, that
```

**Review by georgemccabe — CHANGES_REQUESTED** [review]
These changes look good overall. I like that there are dedicated instructions for creating a coordinated release and steps that were previously found in the release guide for each component are now in a dedicated section so that there is no confusion as to who should complete these tasks.

I made a few minor suggestions that are not required. Feel free to accept or deny them as you feel appropriate.

**Review by georgemccabe — APPROVED** [review]
I approve. Thanks for making those edits.

**Comment by lisagoodrich** [comment]
Hi @georgemccabe,
I need a little more guidance.  I couldn't find the word "Save" anywhere on the page that you linked to.
I completely agree that "Save" should be bolded.  @jprestop, we don't define this in the User's Guide.  I think we should.  What do you think?

> @lisagoodrich, please advise on proposed formatting changes. I changed...
> From Italics: "Click the Save button"
> To Bold: "Click the Save button"
> Per https://github.com/dtcenter/METplus/issues/1667, I'm not sure which it should be. My personal opinion is that bold sticks out better than italics, but please LMK what standard you've already defined, if any.

Since I couldn't find the word "Save" on the main page, I started poking around and found this:
https://metplus--2288.org.readthedocs.build/en/2288/Release_Guide/metplus_official.html#create-release-on-github
The directories listed should be in italics.  I'm not sure if this is within the scope of this pull request or not.  If you point me to the correct page, I'm happy to check formatting.

File: docs/Contributors_Guide/github_workflow.rst
Comment (JohnHalleyGotway) [reply docs/Contributors_Guide/github_workflow.rst:1]: Just updated that. Thanks for finding it.

**Comment by JohnHalleyGotway** [comment]
@lisagoodrich, I'm moving comments over from Slack to a comment on this discussion:

Let me point out the diffs more clearly...

Please compare the formatting:
- In this existing version: https://metplus.readthedocs.io/en/latest/Release_Guide/metplus_official.html#update-dtc-website
- To my pull request version: https://metplus--2288.org.readthedocs.build/en/2288/Release_Guide/metplus_official.html#update-release-notes

The difference is in italics vs bold.

So for anything that's the name of an "entry" on the release page to be filled out, I used bold. And then I enclosed the value in parentheses. For example:
 - Select the **Release Type** (“Recommended” for official or bugfix releases or “Development” for development versions).

And my question is whether or not we've defined the use of italics vs bold vs parentheses for this sort of thing?

**Comment by jprestop** [comment]
>  @jprestop, we don't define this in the User's Guide. I think we should. What do you think?

@lisagoodrich We can definitely talk about adding this sort of information to the Contributor's Guide for bolding.

> Since I couldn't find the word "Save" on the main page, I started poking around and found this:
https://metplus--2288.org.readthedocs.build/en/2288/Release_Guide/metplus_official.html#create-release-on-github
The directories listed should be in italics.

With regard to the above quote @lisagoodrich what directories are you referring to that should be in italics?  Can you please give an example or two so I can find the correct place?

@JohnHalleyGotway I'm going to add a few suggestions on individual lines and will follow up momentarily.

**Comment by jprestop** [comment]
@JohnHalleyGotway Actually, after reviewing, I don't have any suggestions.  I do see some funky formatting with "https://" in this section: https://metplus--2288.org.readthedocs.build/en/2288/Release_Guide/metplus_bugfix.html#create-release-on-github but I don't think you changed anything on that page...

**Comment by lisagoodrich** [comment]
@jprestop, my mistake.  I was skimming and thought the web addresses were directories.  Please ignore.

@JohnHalleyGotway, the formatting looks fine to me.  It looks the same in both versions.  I'm assuming that is what you want.  I only looked at this on the web pages.  Is there somewhere I should be reviewing this in Github?  I'm not sure what to do.
The only feedback I have is, in the User's Guide, we are not using second person (you).  We're keeping it in third person, (the contributor, the user etc.) But I don't know what we'd want to do the same for the Contributor's guide.

**Review by jprestop — APPROVED** [review]

**Comment by lisagoodrich** [comment]
@JohnHalleyGotway, I'm sorry I haven't reviewed anything this big before.  Do I click "Squash and merge" or will this mess things up?

**Comment by jprestop** [comment]
> @JohnHalleyGotway, I'm sorry I haven't reviewed anything this big before. Do I click "Squash and merge" or will this mess things up?

@lisagoodrich Thanks for checking in.  There are still some things here that @JohnHalleyGotway may wish to respond to.  You should not click "Squash and merge".   According to our [Merging Pull Requests](https://metplus.readthedocs.io/en/develop/Contributors_Guide/github_workflow.html#merging-pull-requests) section of the Contributor's Guide: "**As permissions allow, the requestor is responsible for merging the pull request once it has been approved.**" so John should be the one to "Squash and merge".  :)

**Comment by JohnHalleyGotway** [comment]
@lisagoodrich should I be waiting for your approval on this PR? Is this something you're planning to do? Or should I remove you as a reviewer and proceed with the feedback from Julie and George?

**Comment by lisagoodrich** [comment]
@JohnHalleyGotway, @jprestop told me I didn't need to approve it.  So I didn't.  She said only one person needs to approve it and that's normally what you prefer.  If you like, I can find my directions on how to do this and approve it or you can just remove me.

**Pull request merged**
